bin/oref0-get-profile.js

```diff
@@ -42,40 +42,42 @@ function updatePreferences (prefs) {
 
 if (!module.parent) {
     
-    var cwd = process.cwd()
-        
-    var pumpsettings_input = process.argv.slice(2, 3).pop();
-
-    if (pumpsettings_input !== undefined && pumpsettings_input.indexOf('export-defaults') > 0) {
-      exportDefaults();
-      process.exit(0);
-    }
-
-    if (pumpsettings_input !== undefined && pumpsettings_input.indexOf('update-preferences') > 0) {
-      var prefs_input = process.argv.slice(3, 4).pop();
-	  if (!prefs_input) { console.error('usage: ', process.argv.slice(0, 2), '<preferences.json>'); process.exit(0); }
-      var prefs = require(cwd + '/' + prefs_input);
-      updatePreferences(prefs);
-      process.exit(0);
-    }
-
+    var argv = require('yargs')
+      .usage("$0 pump_settings.json bg_targets.json insulin_sensitivities.json basal_profile.json [preferences.json] [--model model.json] [<carb_ratios.json>] [<temptargets.json>]")
+      .option('model', {
+        alias: 'm',
+        describe: "Pump model response",
+        default: false
+      })
+      .strict(true)
+      .help('help')
+
+    var params = argv.argv;
+    var pumpsettings_input = params._.slice(0, 1).pop()
     if ([null, '--help', '-h', 'help'].indexOf(pumpsettings_input) > 0) {
       usage( );
       process.exit(0);
     }
-    
-    var bgtargets_input = process.argv.slice(3, 4).pop()
-    var isf_input = process.argv.slice(4, 5).pop()
-    var basalprofile_input = process.argv.slice(5, 6).pop()
-    var preferences_input = process.argv.slice(6, 7).pop()
-    var carbratio_input = process.argv.slice(7, 8).pop()
-    var temptargets_input = process.argv.slice(8, 9).pop()
-    
+    var bgtargets_input = params._.slice(1, 2).pop()
+    var isf_input = params._.slice(2, 3).pop()
+    var basalprofile_input = params._.slice(3, 4).pop()
+    var preferences_input = params._.slice(4, 5).pop()
+    var carbratio_input = params._.slice(5, 6).pop()
+    var temptargets_input = params._.slice(6, 7).pop()
+    var model_input = params.model;
+    if (params._.length > 6)
```

Comment:
Actually if the `if (params._.length > 6) {...}` was removed it works for me since only the `model` option was added